### ISSUE TITLE
Core/Button not semantic

### ISSUE THREAD

**Issue by sondrebh** [WordPress/gutenberg#18654]
While doing some theming the other day I noticed that the core button is not semantic, even so, not using appearance : button. Is this best practice? 

IMO it should be an actual button tag, with either an <a> tag inside or wrapped around. Or, just an <a> tag alone using appearance: button or something like that.

**Comment by enriquesanchez** [comment]
Hi @sondrebh 

I think there's a naming issue here, but not so much one of functionality.

As I understand it, the core/button is intended to allow users to add links, particularly links for a call to action (Learn more, etc.).  I think the issue comes from what the user expectations are around how those links should look like.

I don't see any a11y issues with the implementation. It's an `a` tag that is properly marked up and can only be used as a link. 

Please let me know if you still think there is a concern and we can reopen the issue.
